doc/UsersGuide/source/tables/fv3_diag_table.rst

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
```suggestion
:orphan:

```
This prevents a warning about the file not being in a table of contents anywhere.